src/config/section/compute.js

```diff
@@ -104,7 +104,8 @@ export default {
           icon: 'reload',
           label: 'label.action.reboot.instance',
           dataView: true,
-          show: (record) => { return ['Running'].includes(record.state) }
+          show: (record) => { return ['Running'].includes(record.state) },
+          args: ['bootintosetup']
```

Comment:
@shwstppr , I should be able to replace this with
```
args: (record) -> { return record.hypervisorType === 'vmware' ? ['bootintosetup'] : [] }
```
am I? It seems not to work. How can I find what to use as parameter of the anonymous function and what attributes it has?

tnx